src/server/systems/NetworkingServer.cpp

```diff
@@ -39,6 +37,14 @@ NetworkingServer::~NetworkingServer() {
         m_stateUdpSocket_t->unbind();
     }
 
+    // disconnect any connected clients
+    // if you don't do this you will get the "failed to bind to socket"
+    // error message when restarting the server.
```

Comment:
fun fact: this still happens

Reply:
can you change this comment then, its misleading